apps/rotm/behaviours/caseworker-email.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+const Emailer = require('hof-behaviour-emailer');
+const path = require('path');
+const moment = require('moment');
+const config = require('../../../config');
+const uuidv1 = require('uuid/v1');
+
+
+const getDataRows = (model, translate) => {
+  const getLabel = key => translate(`email.caseworker.fields.${key}.label`);
+  return [
+      {
+        table: [
+          { label: getLabel('uniqueId'), value: uuidv1() },
+          { label: getLabel('submitted'), value: moment().format(config.dateTimeFormat) },
+          { label: getLabel('source'), value: model.source },
+          { label: getLabel('more-info'), value: model['more-info'] }
```

Comment:
I am not passing in the image label because the html is going to be displayed slightly differently